tests/bluetooth/tester/src/audio/btp_cap.c

```diff
@@ -951,3 +948,9 @@ uint8_t tester_unregister_cap(void)
 {
 	return BTP_STATUS_SUCCESS;
 }
+
+uint8_t cap_extern_ext_ad_setup(void)
+{
+	cap_inhibit_setting_up_ext_ad = true;
+	return 0;
+}
```

Comment:
Could you please add a comment stating the purpose of this function, and why it is needed?

Reply:
added